GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -533,7 +533,8 @@ private bool HandlePushOnExit(ref bool isError, FormProcess form)
                 {
                     if (!form.ProcessArguments.Contains(" -f ") && !form.ProcessArguments.Contains(" --force"))
                     {
-                        Trace.Assert(form.ProcessArguments.StartsWith("push "), "Arguments should start with 'push' command");
+                        // Note that e.g. WSL may add other arguments prior to the actual command so push may not be first.
+                        Debug.Assert(form.ProcessArguments.Contains("push "), "Arguments should start with 'push' command");
```

Comment:
Is the comment still valid?

Reply:
Yes, the extra arguments (see first comment) is mostly hidden for the user